package.json

```diff
@@ -40,18 +42,20 @@
     "mongoose": "^5.0.2",
     "morgan": "^1.5.1",
     "natural": "^0.5.4",
-    "ogr2ogr": "jdesboeufs/ogr2ogr",
+    "ogr2ogr": "^1.3.0",
     "on-finished": "^2.3.0",
     "passport": "^0.4.0",
     "passport-oauth2": "^1.4.0",
     "plunger": "^0.1.3",
+    "pump": "^3.0.0",
```

Comment:
Duplicates with `mississippi` but I prefer explicit dependencies :)

Reply:
Me too, also, `mississippi` is too hard to type.